cross-compiling/automatic_cross_compile.sh

```diff
@@ -22,43 +24,40 @@ source $THIS_DIR/env.sh $TARGET
 bash $THIS_DIR/get_sysroot.sh
 
 # Remove ROS2 old cross-compilation workspace and get a new one
-cd $BASE_DIR
-sudo rm -rf ros2_cc_ws
-mkdir -p ros2_cc_ws/src
-cd ros2_cc_ws
-wget https://raw.githubusercontent.com/ros2/ros2/"$ROS2_DISTRO"/ros2.repos
-vcs import src < ros2.repos
+sudo rm -rf $BASE_DIR/ros2_cc_ws
+bash $THIS_DIR/get_ros2_sources.sh --distro=$ROS2_DISTRO --ros2-path=$BASE_DIR/ros2_cc_ws
 
 # Get HEAD of branch
 HEAD=$(git ls-remote git://github.com/ros2/ros2 "$ROS2_DISTRO" | cut -c1-7)
 
 # Create install directory for the cross-compilation results
-RESULTS_DIR=$BASE_DIR/install/"$ROS2_DISTRO"_"$HEAD"_"$TARGET"
+TARGET_NAME=$(echo "$TARGET" | tr - _)
```

Comment:
Is this substitution needed?

Reply:
You asked me in another comment to have the underscore instead of the dash for the resulting file

Reply:
I thought you were manually setting the names.
In this case just remove it as we can use the dash sorry